### DISCUSSION TITLE
Please, add another high speed serial baurate option that is compatible with OpenWRT.

### DISCUSSION THREAD

**Discussion by WilliamKWusik** [mriscoc/Ender3V2S1#866] [Ideas]
Please, add another high speed serial baurate option that is compatible with OpenWRT (one that is valid). As 250000 is not valid and gives me an error when I try to stty the /dev/ttyUSB0 device. Using the Creality Box with OpenWRT and Octoprint.

Thanks. :-) <3

**Comment by WilliamKWusik** [comment]
I was able to use M575B500 to set to 500000 and it works great. But I need to send that in every boot from the Octoprint server. I wonder if there's a way to fix that speed without a recompile. Otherwise I will just see if I can recompile for my own use and fix the speed. ;-)

**Reply by mriscoc** [reply]
That is a perfect example of use for a [G-code configuration file](https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files) you can rename it to auto0.g for automatic run at boot.

**Reply by WilliamKWusik** [reply]
Wow, didn't know about that, thank you so much. :-) <3 Cheers!

**Reply by WilliamKWusik** [reply]
BTW: I see in the code that it does support 1mega rate, but I couldn't figure out if it's possible to use the M575 command for that.

**Reply by WilliamKWusik** [reply]
Thanks for the auto0.g tip, it works. And I didn't need to change the serial speed in OpenWRT settings. Just that did alone did the track and now OctoPrint can connect at 500k. I still can't figure out how to use 1000k but I'm happy already. ;-) Thanks again. <3 I wonder if I can turn the LCD on/off with Gcode too. Cheers.

**Reply by mriscoc** [reply]
Turn off screen background: [M256](https://marlinfw.org/docs/gcode/M256.html) B0

**Reply by WilliamKWusik** [reply]
Thanks again. I forgot to test that one as I was thinking it wouldn't come back when you pressed the button. But it does!! Yeah. :-) <3 <3  <hug>

**Comment by WilliamKWusik** [comment]
BTW: I have created a FB Group for users who own the Creality Box and whish to use Octoprint instead: https://www.facebook.com/groups/927210251920670
